examples/load_and_reindex_bboxes.py

```diff
@@ -0,0 +1,336 @@
+"""Load and reindex bounding boxes tracks
+==========================================
```

Comment:
I'm unsure about including the term "reindex" in the title of this example. If one reads it out of context, e.g. a user looking at the gallery of examples, it doens't convey much information on what is being done, because "indexing" is a very loaded term that carries many meanings. You are probably referring to the `xarray` method of the same name, which has a very specific meaning, both most people won't be familiar with that. 

We should try to convey the meaning more concretely. Perhaps "upsample" could be used here? That's essentially what we are doing right? We are generating more datapoints in the time dimension. So perhaps: "Load and upsample bounding boxes tracks". 
The tagline (the sentence below the title) can give a bit more info: "Load bounding boxes tracks and upsample them to match the video frame rate". To me that makes it much clearer what this is about, but my view is definitely subjective.

Reply:
I think this is way clearer, thanks!